app/assets/javascripts/ideas.js

```diff
@@ -89,6 +89,8 @@ function createIdea() {
   function editIdea() {
     $('#latest-ideas').delegate('#edit-idea', 'click', function() {
       var $idea = $(this).closest(".idea");
+      @rrgayhart
+      // it says 'Uncaught TypeError: Cannot set property 'contentEditable' of null'
       document.getElementById("idea-title" + $idea.attr('data-id')).contentEditable = true;
```

Comment:
Okay, so checkout what the result of `"idea-title" + $idea.attr('data-id')` looks like. 

Does anything with the id `idea-title1` exist in your document?

Also - can you use the $idea variable that you define earlier for this?